### ISSUE TITLE
switch out eventemitter for eventtarget

### ISSUE THREAD

**Issue by jimmywarting** [node-usb/node-usb#603]
you know... in order to follow the spec more...

**Comment by mildsunrise** [comment]
dropping support for Node.js < 14.5.x just for a detail in the hotplug detection API seems not worth it for me... i'd give more priority to stuff like switching Error for DOMError imho

**Comment by jimmywarting** [comment]
node v20 just got released so support for v14 will end in 2023-04-30
v16 is the LTS now.

**Comment by mildsunrise** [comment]
yeah but this is a big project and y'know. not sure what we're really supporting in practice, but at least in theory we're adamant to drop support for old Nodes. not saying I wouldn't consider it though. @thegecko thoughts?

**Comment by thegecko** [comment]
I agree @mildsunrise. While I'm not adamant we support older node versions forever, I'm also keen we break people as little as possible.

Are there any other benefits we can get through dropping older node support which would make this worthwhile?

**Comment by thegecko** [comment]
I'm wondering if we can at least drop node 10 support and allow to N-API 8 features
